docs-src/go/chapter-durable-execution/retrieve-event-history.md

```diff
@@ -0,0 +1,80 @@
+---
+id: retrieve-event-history
+title: Retrieve a Workflow Execution's Event History
+description: Learn how to retrieve your Workflow Execution's Event History
+sidebar_label: Retrieve Event History
+tags:
+  - go sdk
+  - developer-guide-doc-type
+  - event history
+---
+
+There are a few ways to view and download a Workflow Execution's [Event History](/concepts/what-is-an-event-history).
+We recommend starting off by using either the Temporal CLI or the Web UI to access it.
+
+### Using the Temporal CLI
+
+Use the Temporal CLI's `temporal workflow show` command to save your Workflow Execution's Event History to a local file.
+Run the command from the `/tests` directory so that the file saves alongside the other testing files.
+
+```text
+/backgroundcheck
+    ...
+    /tests
+        | tests.go
+        | backgroundcheck_workflow_history.json
+```
+
+**Local dev server**
+
+If you have been following along with the earlier chapters of this guide, your Workflow Id might be something like "backgroundcheck_workflow".
```

Comment:
I think that:

> be something like "backgroundcheck_workflow".

would be better as:

> be something like `backgroundcheck_workflow`.

Rationale: The Workflow ID is a literal string value that is specified in a command and/or in code, so rendering it in monospaced seems appropriate. Also, having the period outside the quote contradicts our style (I think...it contradicts [Google style](https://developers.google.com/style/quotation-marks#commas-and-periods-with-quotation-marks), at any rate).

Reply:
Good catch; changed.